src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/Folder.xaml

```diff
@@ -15,13 +15,13 @@
 
   <StringProperty Name="FolderName"
                   Category="Misc"
-                  Description="Name of this folder"
+                  Description="Name of this folder."
```

Comment:
Added trailing periods to descriptions. We already had a test for this, but it wasn't checking all rule files. This PR updates the tests to be more explicit about what rule files are included for each verification.